MetadataExtractor/Formats/Apple/BplistReader.cs

```diff
@@ -123,12 +121,9 @@ static object HandleInt(ref BufferReader reader, byte marker)
 
         static Dictionary<byte, byte> HandleDict(ref BufferReader reader, byte count)
         {
-            var keyRefs = ArrayPool<byte>.Shared.Rent(count);
+            Span<byte> keyRefs = stackalloc byte[count]; // <= 255
```

Comment:
Could we convert the comment into a `Debug.Assert(count <= 255)` instead? It provides a little more safety and is clearer as far as documenting intent.

Reply:
I updated the comment to be clearer (count is an unsigned byte, with a safe range) -- no need for an assert. The comment saves a second when auditing the stackalloc call sites.